src/test/java/org/mule/tools/apikit/output/MuleConfigGeneratorTest.java

```diff
@@ -35,6 +35,8 @@
 
 public class MuleConfigGeneratorTest {
 
+  public static final boolean SHOW_CONSOLE = true;
+  public static final boolean HIDE_CONSOLE1 = false;
```

Comment:
Get rid of the 1 or remove the constants if no needed after refactoring the tests.